### PR TITLE
Change README and add module descriptions

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,190 @@
...
+
+Verify the installation process by creating a sample playbook which invokes
+all dependencies:
+
+```bash
+(ansiblevenv) $ vim opentelekomcloud.yaml
+
+# opentelekomcloud.yaml
+
+- hosts: localhost
```
Comment (lego963) [inline README.md:159]: you can use \`\`\`yaml against \`\`\`bash for yaml files. Here is examples:
```yaml
- hosts: localhost
  tasks:
    - name: Get NAT gateway info
      opentelekomcloud.cloud.nat_gateway_info:
        cloud: otc
      register: gw
  
    - name: debug configs
      debug:
        var: gw.nat_gateways
```

```bash
- hosts: localhost
  tasks:
    - name: Get NAT gateway info
      opentelekomcloud.cloud.nat_gateway_info:
        cloud: otc
      register: gw
  
    - name: debug configs
      debug:
        var: gw.nat_gateways
```

**Review by gtema — CHANGES_REQUESTED** [review]
I am not sure about the way.
1) docs should be auto-generated and not manually
2) not sure publishing will work as should

**Comment by tischrei** [comment]
> I am not sure about the way.
> 
> 1. docs should be auto-generated and not manually
> 2. not sure publishing will work as should

related to 1: These docs are auto-generated. We have to implement this in Zuul as well.
related to 2: I oriented at several collections which are using this procedere. So I thought this is a fast way to have those infos online. There is a much better solution: but I think we need someone who is the owner of our collection cloud. There are collections which are marked as "These are the collections with docs hosted on docs.ansible.com." which you can find here: https://docs.ansible.com/ansible/2.10/collections/index.html For these collections the modules are documented directly in Ansible Galaxy and this is clickable.

I used the way of VMware: https://galaxy.ansible.com/community/vmware to get something online as fast as possible.

File: README.md (hunk 1 above)
Comment (tischrei) [reply README.md:161]: Thanks, I changed it.

**Comment by gtema** [comment]
> related to 2: I oriented at several collections which are using this procedere. So I thought this is a fast way to have those infos online. There is a much better solution: but I think we need someone who is the owner of our collection cloud. There are collections which are marked as "These are the collections with docs hosted on docs.ansible.com." which you can find here: https://docs.ansible.com/ansible/2.10/collections/index.html For these collections the modules are documented directly in Ansible Galaxy and this is clickable.
> 
> I used the way of VMware: https://galaxy.ansible.com/community/vmware to get something online as fast as possible.

This is unfortunately absolutely not documented by Ansible. OpenStack collection is listed there while not having docs in source, on the other side if on the page you refer you switch to the "devel" version - it is gone. So, there is no documented instruction of Ansible Galaxy what to do with docs, so just generating them from source and keeping __in__ the source is not really a correct and future-proof way.

**Comment by tischrei** [comment]
> This is unfortunately absolutely not documented by Ansible. OpenStack collection is listed there while not having docs in source, on the other side if on the page you refer you switch to the "devel" version - it is gone. So, there is no documented instruction of Ansible Galaxy what to do with docs, so just generating them from source and keeping **in** the source is not really a correct and future-proof way.

Yes I know, let us start with "my" solution and meanwhile I try to get infos out of the Ansible IRC or somewhere else, oki?

**Comment by gtema** [comment]
> Yes I know, let us start with "my" solution and meanwhile I try to get infos out of the Ansible IRC or somewhere else, oki?

- CI is not really capable to auto-generate doc and commit it into the source. Generating docs and publish them somewhere - no problem, but not add it into the sourcecode.
- Until this is automat-able any change in the module must include "manual" update in the doc - not practical. It will cause mismatches.
- I guess in the next future we will not achieve docs under docs.galaxy.com. It tend only to include documentation for collections underneath ansible-collections NS (or only what is part of ansible-base), which we are not.

**Comment by tischrei** [comment]
Okay, I removed everything which was not accepted by Artem in leave the README changes to provide a better "get started".

**Review by lego963 — DISMISSED** [review]

**Comment by tischrei** [comment]
recheck

**Review by lego963 — APPROVED** [review]

**Pull request merged**